atlas-content-modeler/tutorial/models/fields/relationship/introduction.mdx

```diff
@@ -0,0 +1,32 @@
+---
+slug: /content-engine/tutorial/models/fields/relationship/introduction
+title: Content Engine Fields - Relationship
+description: Learn how to use Relationship Fields in Content Engine.
+---
+
+**Note**: These tutorials assume that you have already [installed the plugin](../../introduction.mdx).
+
+## Relationship Field
+
+The media field is used to include images and other files.
+
+### Steps to Use the Relationship Field
+---
+1. After navigating to a model, click the + button or if it's the first field, click on the Relationship option
+
+    ![Content Engine sync](../../../../images/Screen_Shot_2021-09-01_at_12.17.53_PM.png)
+
+1. Choose the Relationship Field type
+
+1. Fill in the form fields to fit your needs
+
+    ![Content Engine sync](../../../../images/Screen_Shot_2021-09-01_at_12.18.18_PM.png)
+
+1. Click Create when done to save the field to the model
+
+1. Click Cancel to stop field creation
+
+## Feedback
+---
+- **Questions?** Email us ce-beta@wpengine.com
```

Comment:
Is `ce-beta@wpengine.com` the correct support email for Content Modeler?

Reply:
I don't think so. Looks like a CE thing. We don't have an official feedback/support email. We're planning to handle that through GitHub and wp.org forums.